### PR TITLE
Feat: Added Authentication

### PR DESCRIPTION
**Pull request by deepak29-git**
1.Added Login Page
2 Sign up Page

https://user-images.githubusercontent.com/88363801/154951293-b30eee02-2b01-44a8-bd3d-d316ba3ad86e.mp4

### CODE REVIEW

File: Athentiaction/Login/login.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
+.login {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 90vh;
+  }
+  
+  .login-container {
```
Comment (shashidwivedi) [inline Athentiaction/Login/login.css:8]: Indentation is not proper. You can use prettier.

File: Athentiaction/Login/login.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                <span class="forgot-pass">Forgot your Password?</span>
+              </a>
+            </div>
+          </div>
+  
+          <div class="center">
+            <button class="my-2 login-btn">Login</button>
+          </div>
+          <div class="center">
+            <a class="black-text" href="/Athentiaction/Signup/signup.html">Create New Account</a>
```
Comment (shashidwivedi) [inline Athentiaction/Login/login.html:39]: Authentication spelling is incorrect.

**Review by shashidwivedi — APPROVED** [review]
The rest looks good.

File: Athentiaction/Login/login.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  }
+  
+  .login-btn:hover {
+    background-color: var(--primary-hover);
+  }
+  
+  .center {
+    display: flex;
+    justify-content: center;
+  }
```
Comment (ttahm3d) [inline Athentiaction/Login/login.css:27]: You can create this class in your component library and use it across all other projects.

File: Athentiaction/Login/login.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    <link rel="stylesheet" href="/variable.css">
+    <title>Login</title>
+</head>
+<body>
+    <main class="login">
+        <div class="login-container mt-2">
+          <h2 class="center-text">Login</h2>
+          <div class="input-group">
+            <label class="form-label">Email address</label>
+            <input class="form-control" type="text" />
```
Comment (ttahm3d) [inline Athentiaction/Login/login.html:19]: Though you are using label and input box they aren't connected. Do like below to connect them. Label's `for` attribute's value should be same as the `id` of the input element.

```
 <label class="form-label" for"email">Email address</label>
<input class="form-control" id="email type="email" />
```

**Review by ttahm3d — COMMENTED** [review]
Left few comments. Please take a look.

**Review by ttahm3d — APPROVED** [review]

**Review by neetukumari4858 — APPROVED** [review]
it's Looking good just do some changes which other said .

File: Athentiaction/Login/login.css (hunk 3 above)
Comment (deepak29-git) [reply Athentiaction/Login/login.css:27]: ok

**Pull request merged**